src/cljs/rems/fields.cljs

```diff
@@ -271,7 +273,20 @@
       {:id id
        :type :button
        :on-click (fn [e] (.click (.getElementById js/document upload-id)))}
-      (text :t.form/upload)]]))
+      (text :t.form/upload)]
+     [:a.application-search-tips.btn.btn-link.collapsed
```

Comment:
Doesn't seem like the content is focused after opening. Comparing to the search info.